axelor-base/src/main/java/com/axelor/utils/service/TranslationServiceBaseImpl.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Axelor Business Solutions
+ *
+ * Copyright (C) 2005-2024 Axelor (<http://axelor.com>).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+package com.axelor.utils.service;
+
+import com.axelor.apps.base.service.LocaleService;
+import com.axelor.apps.base.service.user.UserService;
+import com.axelor.meta.db.repo.MetaTranslationRepository;
+import com.google.inject.Inject;
+
+public class TranslationServiceBaseImpl extends TranslationServiceImpl
+    implements TranslationBaseService {
```

Comment:
extending a service from AOP could prevent other modules to extend it, so if we can avoid it, it's better:
- Please do not extend TranslationServiceImpl but instead inject it
- rename this class `TranslationBaseServiceImpl`